tests/lib/rook/base.py

```diff
@@ -81,8 +81,9 @@ def _get_charts(self):
 
     def _get_helm(self):
         url = "https://api.github.com/repos/helm/helm/releases/latest"
-        version = requests.get(url).json()["tag_name"]
-        version = "v3.9.0-rc.1"
+        r = requests.get(url)
+        r.raise_for_status()
+        version = r.json()["tag_name"]
```

Comment:
This looks reasonable but according to the patch a25c502a it was needed by some reason the latest helm version, however it is hardcoded to particular version, so maybe just always downloaded given and tested version of helm instead querying from api.
Interesting that official tool to get helm does not use GitHub api, but parses it from the web UI:
https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3

Reply:
I think using the latest feature release would make the most sense for CI pipeline.

https://github.com/SUSE/rookcheck/commit/a25c502af22f2d375c27bbb8a0dee288ca082cb1 required version `v3.9.0` or greater, but [SLES 15 SP4 Subpackages Updates](https://packagehub.suse.com/packages/helm/) is the only release that provides this requirement?